src/filecoin/FilecoinEngine.ts

```diff
@@ -153,29 +163,40 @@ export class FilecoinEngine extends CurrencyEngine<
   }
 
   async getMaxSpendable(spendInfo: EdgeSpendInfo): Promise<string> {
-    let previousNetworkFee = '0'
-    let spendableBalance = spendInfo.spendTargets[0].nativeAmount ?? '0'
-    let maxTries = 10
+    const publicAddress = spendInfo.spendTargets[0].publicAddress
+
+    if (publicAddress == null) throw new Error('Missing publicAddress')
 
     // Enable skip checks because we don't want insufficient funds errors
     spendInfo.skipChecks = true
 
-    // Continuously query networkFees using `makeSpend` until `tx.networkFee`
-    // is stable before returning the `spendableBalance` amount.
-    // This allows us to do a double-check on the network fee.
-    while (maxTries-- > 0) {
-      spendInfo.spendTargets[0].nativeAmount = spendableBalance
-      const tx = await this.makeSpend(spendInfo)
+    // Probe for an amount to use in our transaction:
+    const txForAmount = await this.makeSpend(spendInfo)
```

Comment:
makeSpend checks for a null spendTarget nativeAmount so we should make sure it has a value before calling it